filesystem/profile.000-msys2.sh

```diff
@@ -15,8 +15,9 @@ _warn_deprecated_winver()
     winver="$(uname -s | sed -ne 's/\([^-]*\)-\([^-]*\).*/\2/p')"
     if [ "$winver" = "6.1" ] || [ "$winver" = "6.2" ]; then
         export __MSYS2_WINDOWS_VERSION_WARNING_DONE="true"
-        echo -e "\e[1;33mThe MSYS2 project is planning to drop active support of Windows 7 and 8.0 sometime during 2022.\e[1;0m" 1>&2;
-        echo -e "\e[1;33mFor more information visit https://www.msys2.org/docs/windows_support\e[1;0m" 1>&2;
+        printf "\e[1;33mThe MSYS2 project is planning to drop active support of Windows 7\e[1;0m\n" 1>&2;
```

Comment:
I'm not really understanding why 'printf' is better for 'dash'.  Did you intend to use '/usr/bin/printf' to get proper processing of the escape sequences?

Reply:
I ran it through shellcheck and it complained that dash doesn't support "-e". Do you have a better idea?

Reply:
Not familiar with spellcheck but yeah, dash doesn't seem to support `-e`.  I would personally remove the escape sequence for baseline stuff and simply print the message but you could either use `/bin/printf` to get the `\e` processing (which I'm guessing was your initial thought) or simply embed the escape sequence directly (e.g. use CTL-V-ESC in 'vi' to get the escape).

I updated my pull request (I think) with the embedded escape sequence; if that's not agreeable then I would simply replace the `printf` with `/bin/printf` or `/bin/echo -e` to bypass the builtins.

I'd also say, I don't personally have any issue with 'dash' being uglier (i.e. outputting the raw `-e \e[1;33mThe MSYS2 project is planning to drop active support of Windows 7 and 8.0 sometime during 2022.\e[1;0m`).

Reply:
PS: Sorry, I also changed the tests to a single test with `-o` rather than two tests.  Pedantic and probably only historically relevant (i.e. when `test` used to execute a `/bin/test`).  Ignore, as you see fit.

Reply:
Nope; don't think it updated the pull-request; it's on the branch though [..]

https://github.com/ah64316/MSYS2-packages/tree/patch-1

[..] although, in hindsight, if I had a project that was important I'd probably prefer to use `\e` rather than embedding non-printable characters.